### PR TITLE
updated json file and link

### PR DESCRIPTION
**Pull request by Chamberezigbo**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4914"></a>

### CODE REVIEW

File: data/chamberezigbo.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "Chamberlain Ezigbo",
   "type": "personal",
-  "bio": "JavaScript Engineer | Front-end Developer",
-
+  "bio": "JavaScript Engineer | Full-Stack Developer",
+  "avatar": "https://github.com/chamberezigbo.png",
```
Comment (kumarsonsoff3) [inline data/chamberezigbo.json:5]: As of our latest update, we no longer need the `avatar` key, so please remove this one. 

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Awesome, it looks better than the last one. 

But your filename should be exact same as your username. So please update it to `Chamberezigbo.json`

Let us know if you need any help with this! :)

**Review by devvsakib — CHANGES_REQUESTED** [review]
please rename your file as your GitHub username, C must be uppercase as your username

**Review by Chamberezigbo — COMMENTED** [review]
Change the json file name from c to C

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**